pkg/distributor/distributor.go

```diff
@@ -455,6 +456,11 @@ func (d *Distributor) Push(ctx context.Context, req *client.WriteRequest) (*clie
 			removeLabel(labelName, &ts.Labels)
 		}
 
+		if mrc := d.limits.MetricRelabelConfigs(userID); len(mrc) > 0 {
```

Comment:
**question**: Is there any instrumentation or debug logging that could be added that would be helpful for operators/users when working with this feature?

Reply:
distributor_received_samples_total and distributor_samples_in_total covers that.